pkg/zpdb/eviction_controller.go

```diff
@@ -23,6 +29,8 @@ const (
 	logAllowMesg = "pod eviction allowed"
 )
 
+var _ controller.ZPDBEvictionController = (*EvictionController)(nil)
```

Comment:
I wouldn't bother with this assertion here - if we break the expectation that `EvictionController` satisfies the `controller.ZPDBEvictionController` interface, then rollout-operator as a whole will still fail to compile, and removing this assertion also removes a dependency between the two packages.